### PR TITLE
Fix parsing values in text protocol containing spaces or newlines

### PR DESCRIPTION
**Pull request by imrivera**
Hi!

I've updated the text protocol worker, so it correctly parses values containing spaces or newlines. I also added a SUITE for this module following the structure of the binary one.

### CODE REVIEW

**Comment by miriampena** [comment]
👋

**Comment by imrivera** [comment]
:wave: Hi Miriam!

Great work in this app. We are going to start using it :)

**Comment by imrivera** [comment]
Sorry, I forced a push in my branch, I rebased the changes on your latest master commit. I also removed a couple of `binary:copy` calls.

Do you think this could be merged into your master?

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+REBAR=rebar
```
Comment (elbrujohalcon) [inline Makefile]: Please don't re-add a `Makefile`. Particularly not one using the old `rebar`. We use `rebar3` directly to maintain this project now.

File: test/mero_SUITE.erl [hunk 2]
```diff
@@ -332,16 +332,20 @@ multiget_defineds(_Conf) ->
...
     %% 13, 14 and 15 will go to the same server
     %% 11, 12 and 16 to a different one
-    Expected = [{<<"17">>, undefined},
-                {<<"15">>, <<"2">>},
-                {<<"14">>, <<"2">>},
-                {<<"13">>, <<"1">>},
-                {<<"16">>, <<"3">>},
-                {<<"12">>, <<"1">>},
-                {<<"11">>, <<"1">>}],
+    Expected = lists:sort([{<<"17">>, undefined},
```
Comment (elbrujohalcon) [inline test/mero_SUITE.erl]: You could've just sorted the list in place, instead of adding a `lists:sort/1` call ;)

File: test/mero_SUITE.erl [hunk 3]
```diff
@@ -353,17 +357,18 @@ multiget_defineds_clustered_keys(_Conf) ->
     ?assertMatch({ok, 3}, mero:increment_counter(cluster, {<<"3">>, <<"16">>})),
     %% 13, 14 and 15 will go to the same server
     %% 11, 12 and 16 to a different one
-    Expected = [{<<"17">>, undefined},
+    Expected = lists:sort(
```
Comment (elbrujohalcon) [inline test/mero_SUITE.erl]: Same as above

File: test/mero_wrk_tcp_txt_SUITE.erl [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+
+
+test_response_parsing(Buffer, ExpectedResult, {MemcachedOp, MemcachedOpArgs}) ->
+    %% Reads from the buffer in different chunk sizes, to exercise the buffering done
+    %% on mero_wrk_tcp_binary.  Check that the parsed result is the expected one.
+    lists:foreach(fun(ReadSize) ->
+        FakeNetwork = spawn_link(fun() -> fake_network_recv(Buffer, ReadSize) end),
+        meck:expect(gen_tcp, recv, fun(_, 0, _Timeout) -> network_read(FakeNetwork) end),
+        {ok, Client} = mero_wrk_tcp_txt:connect("localhost", 5000, {?MODULE, stats, []}),
+        %dbg:tracer(),
```
Comment (elbrujohalcon) [inline test/mero_wrk_tcp_txt_SUITE.erl]: I don't think we need to keep this debug calls on the test.

**Comment by imrivera** [comment]
Thank you for the feedback. To avoid cluttering even more this pull-request I'm going to close this one and open a clean new one

**Pull request closed without merging**